packages/components/src/spectrum/picker/Picker.tsx

```diff
@@ -0,0 +1,100 @@
+import { useMemo } from 'react';
+import { Item, Picker as SpectrumPicker } from '@adobe/react-spectrum';
+import { Tooltip } from '../../popper';
+import {
+  NormalizedSpectrumPickerProps,
+  normalizePickerItemList,
+  normalizeTooltipOptions,
+  PickerItem,
+  PickerItemKey,
+  TooltipOptions,
+} from './PickerUtils';
+import { PickerItemContent } from './PickerItemContent';
+
+export type PickerProps = {
+  children: PickerItem | PickerItem[];
+  /** Can be set to true or a TooltipOptions to enable item tooltips */
+  tooltip?: boolean | TooltipOptions;
+  /** The currently selected key in the collection (controlled). */
+  selectedKey?: PickerItemKey | null;
+  /** The initial selected key in the collection (uncontrolled). */
+  defaultSelectedKey?: PickerItemKey;
+  /**
+   * Handler that is called when the selection change.
+   * Note that this is just an alias for `onSelectionChange` for better
+   * consistency with other components.
+   */
+  onChange?: (key: PickerItemKey) => void;
+  /** Handler that is called when the selection changes. */
+  onSelectionChange?: (key: PickerItemKey) => void;
```

Comment:
Perhaps we should mark this as deprecated to encourage usage of `onChange` instead?